### ISSUE TITLE
Select all / Select none for "Track analytics for user roles" and "Show stats dashboard to additional user roles"

### ISSUE THREAD

**Issue by masteradhoc** [plausible/wordpress#170]
Hi there

I think could be a nice addition in plausible to make a small option below all the checkboxes for the both noted sections with a link to select all / select none (deselect all). This would lead us to be faster in setting up the plugin on each of our agency sites :)

Thanks for considering

**Comment by Dan0sz** [comment]
Do you have an example of where you'd have to check, let's say, more than 5 boxes on either options?

@metmarkosaric, what do you think?

**Comment by masteradhoc** [comment]
@Dan0sz easily have sites with 10 custom roles 😅

**Comment by Dan0sz** [comment]
Okay, I guess I just couldn't imagine wanting to share your stats with or track all those user roles.

Still, @metmarkosaric has to pull the trigger on this one. 😁

**Comment by masteradhoc** [comment]
@Dan0sz yeah that part not really needed for a lot sites. i think for 2-3 i used that. 

But the Track analytics for user roles part is more important. but i guess would be consistent to just do it for both then :)

**Comment by metmarkosaric** [comment]
Thanks! Could be something to consider in the future

**Comment by masteradhoc** [comment]
> Thanks! Could be something to consider in the future

Hey @metmarkosaric - we're in the future now ;) 1.5 years have passed.
Any chance this can be implemented in the near future?

Thanks

**Comment by Dan0sz** [comment]
Hi @masteradhoc,

Spent some hours working on this today, but unfortunately, as the Settings screen is currently built, it's not as simple as I'd hoped.

For now, I saved my progress in https://github.com/plausible/wordpress/pull/268 and will circle back to it later.

**Comment by Dan0sz** [comment]
Hi @masteradhoc,

The select all toggle is merged to master and will be included in the next release!
